package.json

```diff
@@ -8,41 +8,29 @@
     "@fortawesome/free-regular-svg-icons": "^6.4.0",
     "@fortawesome/free-solid-svg-icons": "^6.4.0",
     "@fortawesome/react-fontawesome": "^0.2.0",
-    "@testing-library/jest-dom": "^5.16.5",
-    "@testing-library/react": "^14.0.0",
-    "@testing-library/user-event": "^14.4.3",
     "axios": "^1.4.0",
-    "bootstrap": "^5.1.3",
-    "eslint-config-standard": "^17.1.0",
     "fontawesome": "^5.6.3",
     "i18next": "^23.2.8",
     "i18next-browser-languagedetector": "^7.1.0",
-    "json2csv": "^5.0.6",
-    "moment": "^2.29.4",
     "react": "^18.2.0",
-    "react-bootstrap": "^2.8.0",
     "react-dom": "^18.2.0",
     "react-dotenv": "^0.1.3",
     "react-i18next": "^13.0.2",
-    "react-input-mask": "^2.0.4",
     "react-modal": "^3.16.1",
     "react-router": "^6.14.1",
     "react-router-dom": "^6.14.1",
     "react-scripts": "^5.0.1",
-    "storybook": "^7.0.26",
     "styled-components": "^6.0.3",
     "swiper": "^10.0.4",
     "validator": "^13.9.0",
     "web-vitals": "^3.4.0",
     "yarn": "^1.22.19"
   },
   "scripts": {
-    "start": "npm run build && react-dotenv && react-scripts start",
```

Comment:
Alterei este comando pois:
* Não precisamos fazer build para rodar em modo dev, isso atrasa o início do servidor de desenvolvimento localmente 
* Está usando npm ao invés de yarn